### PR TITLE
chore: improve route list

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

- Use an accordion to show the routes, as the button no longer makes sense as it isn't co-located
- Adds loading skeletons to quotes

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

Extends upon a conversation in general to improve this display.

## Risk

Small risk of a UI regression on the trade widget.

## Testing

Confirm that we can still expand the quotes section and click between them.

### Engineering

☝️

### Operations

☝️

## Screenshots (if applicable)

image

### CODE REVIEW

**Comment by 0xApotheosis** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5163** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5163" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5163?utm_source=stack-comment).

File: src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuote.tsx [hunk 1]
```diff
@@ -26,51 +35,52 @@ import { useAppDispatch, useAppSelector } from 'state/store'
...
   isBest: boolean
   quoteData: ApiQuote
   bestBuyAmountBeforeFeesCryptoBaseUnit: string
+  isLoading: boolean
 }
 
 /*
  TODO: Add loading skeleton - the below is an implementation for when trade quotes had separate loading states.
  They are now unified.
  */
```
Comment (woodenfurniture) [inline src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuote.tsx:49]:
```suggestion
```

File: src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx [hunk 2]
```diff
@@ -312,9 +304,32 @@ export const TradeInput = memo(() => {
...
+            <AccordionItem>
+              <h2>
+                <Center>
+                  <AccordionButton>
+                    <Box as='span' flex='1' textAlign='left'>
+                      <Text
+                        translation={'trade.availableRoutes'}
+                        fontWeight={'semibold'}
+                        color='text.subtle'
+                        _hover={{ color: hoverColor }}
```
Comment (woodenfurniture) [inline src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx:318]: this needs a useMemo

File: src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuotes.tsx [hunk 3]
```diff
@@ -38,14 +39,12 @@ export const TradeQuotes: React.FC<TradeQuotesProps> = memo(({ isOpen, sortedQuo
...
+    [activeQuoteIndex, bestQuoteData?.quote?.steps, isLoading, sortedQuotes],
   )
 
   return (
-    <Collapse in={isOpen}>
-      <Flex flexDir='column' gap={2} width='full' px={4} py={2}>
-        {quotes}
-      </Flex>
-    </Collapse>
+    <Flex flexDir='column' gap={2} width='full' px={4} py={2}>
```
Comment (0xdef1cafe) [inline src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuotes.tsx:46]:
```suggestion
    <Flex flexDir='column' gap={2} width='full' px={0} py={2}>
```

Comment (0xApotheosis) [reply src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuotes.tsx]: https://github.com/shapeshift/web/pull/5163/commits/54ebb1c6bc2cb861bccabf96e8e3906f49439c0d

File: src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx (hunk 2 above)
Comment (0xApotheosis) [reply src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx]: https://github.com/shapeshift/web/pull/5163/commits/54ebb1c6bc2cb861bccabf96e8e3906f49439c0d

File: src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuote.tsx (hunk 1 above)
Comment (0xApotheosis) [reply src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuote.tsx]: https://github.com/shapeshift/web/pull/5163/commits/54ebb1c6bc2cb861bccabf96e8e3906f49439c0d

**Comment by 0xdef1cafe** [comment]
updated screenshot in description and stamped

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**